kvmagent/kvmagent/plugins/deip.py

```
@@ -419,7 +416,7 @@ def create_perf_monitor():
             cidr = None
             vnic_ip = netaddr.IPAddress(NIC_IP)
             for l in o.split('\n'):
-                l = l.strip(' \t\n\r')
+                l = l.strip()
```

Ruff (0.15.5)

[error] Ambiguous variable name: `l`

(E741)